py/embedded-server/README.md

```diff
@@ -0,0 +1,33 @@
+# Deephaven Python Embedded Server
+
+Embeds the Deephaven Core server into a python module, so that it can be started from python, with tables produced
+directly as part of startup.
+
+## Dev envirnoment setup
+Java 11 and Docker are required to build this project, as most of the repository is needed to properly build it.
+Note that jpy or deephaven-jpy (built for your OS and archetecture) and the deephaven server apiwheel is also
+required. 
+
+## Build
+From the root directory of this repository
+```shell
+$ ./gradlew :py-embedded-server:assemble
+```
+
+## Install
+```shell
+$ pip install py/embedded-server/build/wheel/deephaven_server-0.12.0-py3-none-any.whl
+```
+
+## Quick start
+
+```python
+>>> from deephaven_server import *
+>>> s = Server()
```

Comment:
Did this get worked out with the other local oddities you were hitting (I think they boiled down to jpy issues)?

Reply:
I did get this to work w/ the jpy-consortium built 3.10 wheel. The one change I had to make:

```shell
$ rm /tmp/venv-3.10/lib64/python3.10/site-packages/jpyconfig.py
```

Otherwise I would get the error:
```
>>> s = Server()                                                                                                                                                                                                                                                              
Traceback (most recent call last):                                                                                                                                                                                                                                            
  File "<stdin>", line 1, in <module>                                                                                                                                                                                                                                         
  File "/tmp/venv-3.10/lib64/python3.10/site-packages/deephaven_server/server.py", line 27, in __init__                                                                                                                                                                       
    start_jvm(jvm_args=jvm_args)                                                                                                                                                                                                                                              
  File "/tmp/venv-3.10/lib64/python3.10/site-packages/deephaven_server/start_jvm.py", line 79, in start_jvm                                                                                                                                                                   
    jpyutil.init_jvm(                                                                                                                                                                                                                                                         
  File "/tmp/venv-3.10/lib64/python3.10/site-packages/jpyutil.py", line 431, in init_jvm                                                                                                                                                                                      
    cdll = preload_jvm_dll(jvm_dll_file=jvm_dll,                                                                                                                                                                                                                              
  File "/tmp/venv-3.10/lib64/python3.10/site-packages/jpyutil.py", line 352, in preload_jvm_dll                                                                                                                                                                               
    return ctypes.CDLL(jvm_dll_file, mode=ctypes.RTLD_GLOBAL)                                                                                                                                                                                                                 
  File "/usr/lib64/python3.10/ctypes/__init__.py", line 374, in __init__                                                                                                                                                                                                      
    self._handle = _dlopen(self._name, mode)                                                                                                                                                                                                                                  
OSError: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/8.0.332-9/x64/jre/lib/amd64/server/libjvm.so: cannot open shared object file: No such file or directory
```

It seems like we should *not* produce this file in the artifact b/c users' env won't match the build env.

Reply:
I'm not following - which file shouldn't be produced?

Reply:
The jpy wheel contains `jpyconfig.py`, which points to the java home that was used in CI.

Reply:
Hmm, I don't get that file installed for whatever reason, i'll take a closer look at what we produce. I only build the jpy wheel from inside docker, copy it out to the host, and run it there (and with a different jdk version), and it seems to behave correctly, since we call into jpyutil before trying to start up jpy itself. That said I'm using deephaven_jpy, perhaps it makes a difference?

Reply:
```
$ unzip -l py/jpy/build/deephaven-jpy-wheel/deephaven_jpy-0.12.0-cp37-cp37m-linux_x86_64.whl 
Archive:  py/jpy/build/deephaven-jpy-wheel/deephaven_jpy-0.12.0-cp37-cp37m-linux_x86_64.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
    36224  2022-05-25 22:45   jdl.cpython-37m-x86_64-linux-gnu.so
   502840  2022-05-25 22:45   jpy.cpython-37m-x86_64-linux-gnu.so
    25344  2022-05-25 22:44   jpyutil.py
    11358  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/LICENSE
     7993  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/METADATA
      910  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/NOTICE
      104  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/WHEEL
       16  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/top_level.txt
      771  2022-05-25 22:46   deephaven_jpy-0.12.0.dist-info/RECORD
---------                     -------
   585560                     9 files
```

Reply:
Yeah, this might be a "my" issue, or something that we've already fixed in deephaven-core that we should probably fix on jpy-consortium. Something to look out for once we use the public wheels.